src/thimble_server/data_access_layer/post.clj

```diff
@@ -25,3 +39,17 @@
     (if (and (user-exists? poster)
              (= (insert-post! postid poster is-original-int) 1))
         postid nil)))
+
+(defn add-file-to-post!
+  "takes audio and post id
+   calls file handler
+   updates table so post row has filekey in entry
+   @param   audio  [blob]    the audio file
```

Comment:
blob is not a datatype. this is more likely to be a BufferedInputStream or some similar Java class. even just putting a "?" for now would be fine.